SQL/2015-06-11-subprojectConfigTable.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE subproject (
```

Comment:
No, the subproject table is for defining the subprojects (and taking that out of the config.xml), not the relations between subprojects and projects.